website/docs/basics/blocks-special-use-cases.md

```diff
@@ -6,5 +6,84 @@ sidebar_label: Special Use Cases
+
+But now you have some default attributes set in that image block that you don't want to be set if the block is added using the innerBlock in the carousel. You can change that using our [overrideInnerBlockAttributes](helpers-javascript#overrideinnerblockattributes) helper. Just add this helper to your block entry point, add the attributes you want to change, and you are all set. 
+
+Keep in mind that this will be applied to all innerBlocks inside your carousel block. If you want to limit this to only a specific block, you will need to fiddle around with that block's conditions (you can get that specific block name from the props).
+
+Next time you add your image to the carousel, it will automatically change those attributes.
+
+> Important note: If you for some reason copy the inner block from the carousel to the top-level editor, this helper will not fire and not change back to original attributes. You have to manually remove those attributes from the Code editor in the admin.
+
+**Implementation**
+
+Here you can see our example of this helper used in the [carousel block](https://github.com/infinum/eightshift-frontend-libs/blob/develop/blocks/init/src/Blocks/custom/carousel/carousel-block.js).
+
+## Provide simple wrapper to all innerBlock
+
+**Usage:**
+- Change attributes in the block editor and in the frontend.
+- Set attributes in the database.
+
+**Use Case:**
+
+All of our blocks come with the wrapper implemented, and this is awesome, but sometimes you just don't want to use the wrapper, and you want only to use the simple version of the wrapper. You can provide the attributes overrides in your block, and you are good to go.
+
+But if you need to change attributes in the block added as an innerBlock, you have a little problem. You can use the same helper as we used in the previous chapter, or you can use our abstraction of that helper called [overrideInnerBlockSimpleWrapperAttributes](helpers-javascript#overrideinnerblocksimplewrapperattributes).
+
+This helper will set wrapper to simple in all your innerBlocks.
+
+**Implementation**
+
+Here you can see our example of this helper used in the [column block](https://github.com/infinum/eightshift-frontend-libs/blob/develop/blocks/init/src/Blocks/custom/column/column-block.js).
+
+## Override innerBlock attributes only on the frontend
+
+**Usage:**
+
+- Change attributes on the frontend only.
+- Don't set attributes in the database.
+
+**Use Case:**
+
+If you have a block containing inner blocks and only want to change attributes on the front end of the implementation and not in the block editor. But the standard way is really hard because all inner blocks are rendered on the frontend as an HTML string, and you don't have access to any of the attributes before it is rendered.
```

Comment:
```suggestion
If you have a block containing inner blocks and only want to change attributes on the front end of the implementation and not in the block editor. The standard way is really hard because all inner blocks are rendered on the frontend as an HTML string, and you don't have access to any of the attributes before it is rendered.
```